payload-examples/api.github.com/repository.edited.2.payload.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
I wasn't sure what to call this one. Should I name it `repository.edited.with-default_branch-edit.payload.json`

Reply:
yeah I think the more descriptive the better